stories/scss/form/checkbox/checkbox.stories.html

```diff
@@ -1,5 +1,10 @@
-<label class="checkbox">
-	<input class="checkbox-input" type="checkbox" [attr.aria-hidden]="isAriaHidden" [ngModel]="isChecked" (ngModelChange)="toggle($event)"/>
+<label class="checkbox {{ palette }}">
+	<input class="checkbox-input" 
+		type="checkbox" 
+		[attr.aria-hidden]="isAriaHidden" 
```

Comment:
on avait déjà mis un attribut aria-hidden sur un input ? ça me parait bizarre

Reply:
It seems to me that it was to manage the mixed.